gempy/addons/RexCloud_Api_key.txt

```diff
@@ -0,0 +1,4 @@
+4ffb5ad8-2fb7-4bd1-ae4a-646ee850df66
+xwYiAn2Fws07pflRPZQoo5yVMkoreFKM
+# put your API tokens and secrets in the lines above.
+# The key file is not tracked in git so it will not be uploaded to the repository.
```

Comment:
The comment says that this Key file won't be uploaded but it is in this pull request. It seems that the .gitignore file is wrong here.
Is this your personal API key or just a dummy one?

Reply:
You could also include the link https://www.rexos.org/getting-started/ to make it easier for others to generate their own API key

Reply:
yes, I was trying to test that gitignore worked, which is clearly not the case :D